src/main/java/com/mraof/minestuck/util/MSSoundEvents.java

```diff
@@ -53,6 +53,11 @@
 	public static final SoundEvent MUSIC_UNDEAD = getNull();
 	public static final SoundEvent MUSIC_TOWERS = getNull();
 	
+	public static final SoundEvent MUSIC_SICKEST_FIRES = getNull();
+	public static final SoundEvent MUSIC_RANCOROUS_GAMBLIGANT = getNull();
+	public static final SoundEvent MUSIC_WHAT_GOES_UP = getNull();
+	public static final SoundEvent MUSIC_RISE_UP = getNull();
```

Comment:
"Rise up" appears to be unused. That doesn't seem right.

Reply:
The situation it is intended for(god tiering) has yet to be added to the game. However I wanted to avoid potentially losing track of the music before god tiers are added

Reply:
I'm a little bit conflicted about it, but I'll leave it be.

Reply:
I assume the conflict is due to it increasing the size of the project file despite not being used, which I can understand